README.md

```diff
@@ -3,6 +3,16 @@ Receive a notification email every time a new git tag is found in AOSP<br/>
 _Will_ generates a change log between different aosp tags. 
 
 ### Usage
+#### New tag notification
 ```
-$ ./check_for_new_build_label.sh <notification email address>
+$ ./check_for_new_build_label.sh <notification email address> <AOSP working directory>
+```
+
+The param `AOSP working directory` is optional and must specify the *absolute* path of the directory in which will be cloned the whole AOSP code.
```

Comment:
`must specify the *absolute* path of the directory in which the whole AOSP code will be cloned.` sounds better to me